src/xrpld/app/paths/detail/StepChecks.h

```diff
@@ -65,17 +65,18 @@ checkFreeze(
     if (view.rules().enabled(fixFrozenLPTokenTransfer))
     {
         if (auto const sleDst = view.read(keylet::account(dst));
-            sleDst && sleDst->isFieldPresent(sfAMMID))
+            sleDst && sleDst.fsfAMMID().has_value())
```

Comment:
Either prefix the name of optional fields with `o`, or just throw if there's no value.